instrumentation/struts-2/struts-2.gradle

```diff
@@ -0,0 +1,27 @@
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    group = "org.apache.struts"
+    module = "struts2-core"
+    versions = "[2.3.20,)"
```

Comment:
is this the minimum version that this instrumentation works with? if so, then let's rename module to `struts-2.3` instead of my suggestion below of `struts-2.0`

Reply:
2.3.20 from year 2014 is the earliest version of Struts2, that is available for download on Struts official download page: https://archive.apache.org/dist/struts/ I wasn't adventurous enough to search for even earlier versions. :)